app/utils/highcharts.js

```diff
@@ -0,0 +1,31 @@
+import { assign } from '@ember/polyfills';
+import merge from 'deepmerge';
+
+function createChartOptions({
+  chartOptions = {},
+  data = [{ color: '#aaaaaa', data: 0, id: 'placeholder' }],
+  theme = {},
+} = {}) {
+  const themePlusChartOptions = merge(theme, chartOptions);
+
+  return assign({ series: data }, themePlusChartOptions);
```

Comment:
I think, previously, `assign` from `@ember/polyfills` had been used instead of `Object.assign` because we needed to support IE 11.